### PR TITLE
Ruby deps fix

### PR DESCRIPTION
**Pull request by erno**
Gets Docker setup running again.

### CODE REVIEW

**Comment by solita-mattipulkkinen** [comment]
I know the docker setup will work with this change, but there's a .ruby-version file which still declares the version to be 2.7.3, should that be changed as well? And what about vagrant? makeitso.sh also has version 2.7.3 specified.

File: Gemfile.lock [hunk 1]
```diff
@@ -273,6 +273,7 @@ PLATFORMS
 DEPENDENCIES
   compass
   github-pages
+  webrick
```
Comment (petteripoyhtari) [inline Gemfile.lock:276]: Where is this needed?

Comment (petteripoyhtari) [reply Gemfile.lock:276]: Seems to work. Resolved.

**Review by petteripoyhtari — APPROVED** [review]
Looks good and works locally on Mac M3.

**Review by tatut — APPROVED** [review]

**Pull request merged**